app/src/main/java/com/google/android/soundchecker/AudioEncoderDecoderActivity.kt

```diff
@@ -493,6 +498,46 @@ class AudioEncoderDecoderActivity : ComponentActivity() {
                             }
                         }
                     }
+                    Spacer(modifier = Modifier.padding(4.dp))
+                    Row {
+                        Text(text = "Encoder Delay")
+                        Spacer(modifier = Modifier.padding(4.dp))
+                        var expanded by remember { mutableStateOf(false) }
+                        Box(
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .wrapContentSize(Alignment.CenterEnd)
+                        ) {
+                            // Create the dropdown menu
+                            DropdownMenu(
+                                expanded = expanded,
+                                onDismissRequest = { expanded = false },
+                                modifier = Modifier.fillMaxWidth()
+                            ) {
+                                ENCODER_DELAYS.forEachIndexed { index, bin ->
+                                    DropdownMenuItem(
+                                        text = { Text(bin.toString()) },
+                                        onClick = {
+                                            mEncoderDelay =
+                                                ENCODER_DELAYS[index]
+                                            mEncoderDelayText.value =
+                                                ENCODER_DELAYS[index].toString()
+                                            expanded = false
+                                        },
+                                        enabled = mSpinnersEnabled.value
```

Comment:
If the DropDownMenu item is enabled or not is decided by mSpinnersEnabled or not, how about make the DropdownMenu enabled by value of mSpinnersEnabled?